src/routes/(console)/project-[region]-[project]/databases/database-[database]/collection-[collection]/createAttribute.svelte

```diff
@@ -24,14 +24,20 @@
     };
     let error: string;
 
+    async function updateCollectionColumns() {
+        let selectedColumns = preferences.getCustomCollectionColumns(collectionId);
+        selectedColumns.push(key ?? data?.key);
+        await Promise.all([
+            preferences.setCustomCollectionColumns(collectionId, selectedColumns),
+            invalidate(Dependencies.COLLECTION)
```

Comment:
Do we need to invalidate collection if only prefferences for columns changed?
If not, we could remove.
If yes, Promise.all might be doors opened for race condition

Reply:
We need a fresh collection data on create or removal so the its up to date. Promise should be fine. was there before, just grouped them.

Reply:
If we need to invalidate after changign column prefferences, why do it in promise.all?

What if it invalidates first, then saves new prefs - wont that show old stuff instead of new?

Reply:
mm makes sense. removing promise all.